app/src/main/java/com/android/mycax/floatingvolume/services/FloatingVolumeService.java

```diff
@@ -567,4 +574,38 @@ public void onConfigurationChanged(Configuration newConfig) {
     private void getWindowManagerDefaultDisplay() {
         mWindowManager.getDefaultDisplay().getSize(szWindow);
     }
+
+    private void checkBarsSettings() {
+        try {
+            /**/
```

Comment:
If we're going in this direction we also need to be able to disable Voice Call seekbar but that will need more changes within the service for optimisation. Can you please tick "Allow edits from maintainers" so I can add those changes directly to the pull request please?